lib/kindle_highlights/client.rb

```diff
@@ -5,12 +5,12 @@ class Client
 
     attr_reader :books
         
-    def initialize(email_address, password)
+    def initialize(email_address, password, **mechanize_kwargs)
```

Comment:
I've been thinking about changing these to be named parameters, so:

```ruby
def initialize(email_address:, password:, mechanize_options: {})
```

I like the explicitness of saying "these are mechanize options, and we expect a Hash", rather than a splatted arg. What do you think about that?